### DISCUSSION TITLE
Archive Audio language setting

### DISCUSSION THREAD

**Discussion by Brainfighter** [crunchy-labs/crunchy-cli#278] [Q&A]
Hello,
is it possible to set a language as optional ? I would like to download movies in English and have Japanese and German as optional. Meaning download all in English and include Japanese and English if available.

**Comment by bytedream** [comment]
This is default behavior of the `archive` command.
E.g. `crunchy-cli archive -a en-US -a ja-JP -a de-DE ...` only downloads the available languages and skips the rest. It would still download if english does not exist tho

**Reply by Brainfighter** [reply]
Thanks, if it downloads episodes in ja-JP and I run the download again and the episodes are now available, will it re-download the episodes that are now available in English ? I run the folling command:
crunchy-cli -q --user-agent "Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 5.1; Trident/4.0)" archive -a de-DE -a en-US -a ja-JP -s en-US -s de-DE --skip-existing --yes -t 32 -o "/{series_name}/{series_name} - S{season_number}E{episode_number}.mkv" ''' + URL

My plan is the run this on a regular basis to download an update the episodes as they come out and will be available in English

**Comment by Nideapum** [comment]
In lunix I use an alias
```bash
alias crunchy-download='crunchy-cli archive -a es-ES -a ja-JP -s es-ES'
```
